### ISSUE TITLE
[loss] focal loss throwing an error when used in place of ce loss

### ISSUE THREAD

**Issue by tijyojwad** [clara-parabricks/VariantWorks#158]
When the nemo cross entropy loss is replaced with the focal loss call, it throws pytorch errors. The focal loss module should be s drop in replacement for the ce loss
